magento/magento2
# Fix the newsletter confirmation email From

Fix the newsletter confirmation email From on multi-store env





### Description (*)


Bug: the newsletter subscription confirmation email has the wrong email in From field
if the customer is assigned to a non-default store and subscribed/unsubscribed via admin panel

### Related Pull Requests


### Fixed Issues (if relevant)

I didn't find the existing issue

### Manual testing scenarios (*)

1. In admin panel of multi-store Magento, open some customer that assigned to non-default store
2. Go to "Newsletter" tab
3. Select (or unelect if it's selected already) the "Subscribed to Newsletter" checkbox and save the customer

Expected:
The customer receives an email with a store-specific email in From header

Actual:
The customer receives an email with default email in From header

### Questions or comments


### Contribution checklist (*)


### Resolved issues:
1. [x] resolves magento/magento2#34963: Fix the newsletter confirmation email From
